### PR TITLE
gen 2 GA readme update

### PR DESCRIPTION
**Pull request by edwardfoyle**
Updates the repo readme to remove preview wording

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
-# Preview: AWS Amplify's new code-first DX (Gen 2) for building backends
+# AWS Amplify Gen 2 Backend
 
-This next generation of Amplify’s backend building experience lets you author your frontend and backend definition completely with TypeScript, a file convention, and Git branch-based environments, is now available in Preview. To learn more, please visit [AWS Amplify (Gen 2)](https://docs.amplify.aws/gen2/).
+This next generation of Amplify’s backend building experience lets you author your frontend and backend definition completely with TypeScript, a file convention, and Git branch-based environments. To learn more, visit [AWS Amplify (Gen 2)](https://docs.amplify.aws/gen2/).
```
Comment (swaminator) [inline README.md:3]: docs.amplify.aws

File: README.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 
 ## Quickstart
 
-To quickly get started with Amplify (Gen 2), please visit [AWS Amplify (Gen 2) Quickstart](https://docs.amplify.aws/gen2/start/quickstart/).
+To get started with Amplify Gen 2, visit [AWS Amplify Gen 2 Quickstart](https://docs.amplify.aws/gen2/start/quickstart/).
```
Comment (swaminator) [inline README.md:7]: https://docs.amplify.aws/react/start/

File: README.md [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
 
 ```bash
 npm create amplify@latest
 ```
 
+## Tutorials
```
Comment (swaminator) [inline README.md:13]: Remove this section.

**Review by swaminator — CHANGES_REQUESTED** [review]
Comments inline.

File: README.md (hunk 1 above)
Comment (edwardfoyle) [inline README.md:3]:
```suggestion
This next generation of Amplify’s backend building experience lets you author your frontend and backend definition completely with TypeScript, a file convention, and Git branch-based environments. To learn more, visit [AWS Amplify (Gen 2)](https://docs.amplify.aws).
```

File: README.md (hunk 2 above)
Comment (edwardfoyle) [inline README.md:7]:
```suggestion
To get started with Amplify Gen 2, visit [AWS Amplify Gen 2 Quickstart](https://docs.amplify.aws/react/start).
```

File: README.md [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
+
+- [Quickstart](https://docs.amplify.aws/gen2/start/quickstart/)
+- [Set up Amplify Auth](https://docs.amplify.aws/gen2/build-a-backend/auth/set-up-auth/)
+- [Set up Amplify Data](https://docs.amplify.aws/gen2/build-a-backend/data/set-up-data/)
+
```
Comment (edwardfoyle) [inline README.md:18]:
```suggestion
```

**Review by swaminator — APPROVED** [review]
LGTM

**Review by 0618 — APPROVED** [review]

**Pull request merged**